grails-app/assets/javascripts/streama/templates/modal--user.tpl.htm

```diff
@@ -33,8 +33,8 @@
       <div class="col-sm-3">
         <label class="control-label">Roles</label>
       </div>
-      <div class="col-sm-8">
-        <select class="form-control" multiple ng-options="role as role.displayName for role in roles track by role.id" ng-model="user.authorities"></select>
+      <div class="col-sm-8" ng-repeat="role in roles">
```

Comment:
ah, the reason why it wasnt on the right hand side is because you have your ng-repeat on the col-sm-8, which is bad practice. The col-sm-8 should wrap everything that should be on the right side, so please create a nesting div, and it is that div that you want to repeat over